scss/bootscore/_forms.scss

```diff
@@ -6,3 +6,25 @@ Forms
 select {
   @extend .form-select;
 }
+
+
+// Form validation icons
+// Allows validation with CSS variables instead using JS.
+// Used in _wc_checkout.scss and bS Contact Form 7 plugin.
+
+// Valid icon
+@function valid-icon-color($color) {
```

Comment:
As requested https://github.com/bootscore/bootscore/pull/563/commits/dd59b1eeec376db932b56b355bf8c6ad2396c212

This may have a conflict with https://github.com/bootscore/bootscore/pull/568. When merged, I have to check this again.